docs/user/metal/install_ipi.md

```diff
@@ -0,0 +1,264 @@
+# Bare Metal IPI (Installer Provisioned Infrastructure) Overview
+
+Current Status: **Experimental**
+
+This document discusses the installer support for an IPI (Installer Provisioned
+Infrastructure) install for bare metal hosts.  This includes platform support
+for the management of bare metal hosts, as well as some automation of DNS and
+load balancing to bring up the cluster.
+
+The upstream project that provides Kubernetes native management of bare metal
+hosts is [metal3.io](http://metal3.io).
+
+For UPI (User Provisioned Infrastructure) based instructions for bare metal
+deployments, see [install_upi.md](install_upi.md).
+
+## Prerequisites
+
+### Ironic
+
+Currently, the `baremetal` platform requires an existing Ironic environment.
+This will eventually be handled by `openshift-install`, with Ironic being
+deployed onto the bootstrap node. Until then, users of the `baremetal` platform
+should use the
+[openshift-metal3/dev-scripts](https://github.com/openshift-metal3/dev-scripts)
+repository to handle configuration of Ironic, and to launch the installer.
```

Comment:
If I was an openshift/installer maintainer, I could probably handle the requirement to use dev-scripts to set up Ironic, but I'd be totally against a requirement to use dev-scripts to launch openshift-install itself

Reply:
Good point.  I've removed "and to launch the installer".